faker/providers/person/vi_VN/__init__.py

```diff
@@ -0,0 +1,1670 @@
+# coding=utf-8
+
+from __future__ import unicode_literals
+from collections import OrderedDict
+
+from .. import Provider as PersonProvider
+
+
+# I statstic with data from https://github.com/duyetdev/vietnamese-namedb
+
+class Provider(PersonProvider):
+    """Provider for Vietnamese person generator"""
+    formats_female = OrderedDict((
+        ('{{first_name_female}} {{mid_name_female}} {{last_name_female}}',
+         0.75),
+        ('{{first_name_female}} {{last_name_female}}', 0.25),
+    ))
+
+    formats_male = OrderedDict((
+        ('{{first_name_male}} {{mid_name_male}} {{last_name_male}}', 0.75),
+        ('{{first_name_male}} {{last_name_male}}', 0.25),
+    ))
+
+    formats = formats_male.copy()
+    formats.update(formats_female)
+
+    first_names_female = OrderedDict((
+        ('Quỳnh', 0.017834394904458598),
```

Comment:
where did you get the distribution from? The dataset in the source doesn't seem to include any numbers.

Reply:
i use statistic, i have a dataset name of person Vietnamese, because i can generate probability for each its

Reply:
We can't use data that is not public. Either remove the probability, or publish your data and link to it